src/components/shared/MobileMenu/index.tsx

```diff
@@ -83,7 +83,14 @@ export const MobileMenu = ({
   useEffect(() => {
     if (
       companyData != null &&
-      Object.values(ApplicationStatus).includes(companyData.applicationStatus)
+      Object.values([
+        ApplicationStatus.CREATED,
+        ApplicationStatus.ADD_COMPANY_DATA,
+        ApplicationStatus.INVITE_USER,
+        ApplicationStatus.SELECT_COMPANY_ROLE,
+        ApplicationStatus.UPLOAD_DOCUMENTS,
+        ApplicationStatus.VERIFY,
+      ]).includes(companyData.applicationStatus)
```

Comment:
@manojava-gk are you sure that this solved the issue? I dont understand the fix. We are missing a lot of the navigation items and the "fix" is to add ApplicationStatus?

Reply:
Yes @jjeroch. Earlier ApplicationStatus was not having DECLINED and CONFIRMED status. But later it got added but removed only in desktop version. I discussed this with Nidhi also.